LayoutTests/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-self/sharedworker-import.https-expected.txt

```diff
@@ -4,11 +4,11 @@ PASS Content Security Policy: Expects allowed for sharedworker-import to same-ht
 PASS Content Security Policy: Expects allowed for sharedworker-import to same-https origin and no-redirect redirection from https context.
 PASS Content Security Policy: Expects allowed for sharedworker-import to same-https origin and no-redirect redirection from https context.: securitypolicyviolation
 PASS Content Security Policy: Expects blocked for sharedworker-import to cross-https origin and keep-origin redirection from https context.
-PASS Content Security Policy: Expects blocked for sharedworker-import to cross-https origin and keep-origin redirection from https context.: securitypolicyviolation
+FAIL Content Security Policy: Expects blocked for sharedworker-import to cross-https origin and keep-origin redirection from https context.: securitypolicyviolation assert_equals: One violation event should be fired expected 1 but got 0
```

Comment:
Ditto.

Reply:
I'm uploading a revised patch for EWS, but will investigate this and resolve in a follow-up.